annotation-processor/src/main/kotlin/pl/touk/krush/source/TablesGenerator.kt

```diff
@@ -1,56 +1,22 @@
 package pl.touk.krush.source
 
-import com.squareup.kotlinpoet.BOOLEAN
-import com.squareup.kotlinpoet.ClassName
-import com.squareup.kotlinpoet.CodeBlock
-import com.squareup.kotlinpoet.DOUBLE
-import com.squareup.kotlinpoet.FLOAT
-import com.squareup.kotlinpoet.FileSpec
-import com.squareup.kotlinpoet.FunSpec
-import com.squareup.kotlinpoet.INT
-import com.squareup.kotlinpoet.LONG
-import com.squareup.kotlinpoet.ParameterSpec
+import com.squareup.kotlinpoet.*
 import com.squareup.kotlinpoet.ParameterizedTypeName.Companion.parameterizedBy
-import com.squareup.kotlinpoet.PropertySpec
-import com.squareup.kotlinpoet.SHORT
-import com.squareup.kotlinpoet.STRING
-import com.squareup.kotlinpoet.TypeName
-import com.squareup.kotlinpoet.TypeSpec
-import com.squareup.kotlinpoet.asClassName
-import com.squareup.kotlinpoet.asTypeName
 import org.jetbrains.exposed.sql.Column
 import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.Table.PrimaryKey
 import pl.touk.krush.env.TypeEnvironment
-import pl.touk.krush.model.AssociationDefinition
-import pl.touk.krush.model.AssociationType
-import pl.touk.krush.model.ConverterDefinition
-import pl.touk.krush.model.EmbeddableDefinition
-import pl.touk.krush.model.EntityDefinition
-import pl.touk.krush.model.EntityGraph
-import pl.touk.krush.model.EntityGraphs
-import pl.touk.krush.model.EnumType
-import pl.touk.krush.model.IdDefinition
-import pl.touk.krush.model.PropertyDefinition
-import pl.touk.krush.model.Type
-import pl.touk.krush.model.allAssociations
-import pl.touk.krush.model.asObject
-import pl.touk.krush.model.asVariable
-import pl.touk.krush.model.entity
-import pl.touk.krush.model.packageName
-import pl.touk.krush.model.traverse
-import pl.touk.krush.validation.AssociationTargetEntityNotFoundException
-import pl.touk.krush.validation.IdTypeNotSupportedException
-import pl.touk.krush.validation.MissingIdException
-import pl.touk.krush.validation.PropertyTypeNotSupportedExpcetion
-import pl.touk.krush.validation.TypeConverterNotSupportedException
+import pl.touk.krush.model.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.